editor.planx.uk/src/pages/FlowEditor/lib/store/preview.ts

```diff
@@ -379,6 +380,7 @@ export const previewStore: StateCreator<
 
   resumeSession(session: Session) {
     set({ ...session });
+    get().setCurrentCard();
```

Comment:
When resuming, we handle the initial state setup here after getting a `ReconciliationResponse`

Please see https://github.com/theopensystemslab/planx-new/pull/1701 for context.